conf/application.conf

```diff
@@ -124,7 +124,9 @@ microservice {
     }
   }
 }
-
+throttle {
+percentage = 100
+}
```

Comment:
I think the plan was to have set locally a fixed value that does will not cause the current journey tests to fail but allow a new journey test to be added that will not be selected to go through our journey.

Reply:
arbirtrary value for now, but should be set to 0 in next task so only NINOS QQ112200Q are inlcuded